feconf.py

```diff
@@ -673,7 +673,7 @@ def get_empty_ratings():
 TOPIC_STATUS_URL_PREFIX = '/rightshandler/change_topic_status'
 TOPICS_AND_SKILLS_DASHBOARD_DATA_URL = '/topics_and_skills_dashboard/data'
 TOPICS_AND_SKILLS_DASHBOARD_URL = '/topics_and_skills_dashboard'
-TRANSLATION_DATA_PREFIX = '/createhandler/translate'
+VOICEOVER_DATA_PREFIX = '/createhandler/voiceover'
```

Comment:
Keep the alphabetical order. (In general, be aware of the context surrounding the lines you're editing.)

Reply:
Done.